types/react-relay/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-relay 11.0
+// Type definitions for react-relay 13.0
```

Comment:
There's another pr of mine that does this. Ideally we'd want that merged first, or split them here. 

There's been a small not so good update to the current types. That'd I'd want to rectify. 

So action, fork types into a sub folder.

Reply:
> There's been a small not so good update to the current types. That'd I'd want to rectify.

@maraisr is this the PR you are referring to? https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57639 Can you elaborate what the issue is? What else is needed to get it merged?